src/main/java/io/antmedia/streamsource/StreamFetcherManager.java

```diff
@@ -112,7 +112,7 @@ public Result startStreaming(Broadcast broadcast) {
 		Result result=new Result(false);
 
 		try {
-			StreamFetcher streamScheduler = streamFetcherFactory.make(broadcast, scope);
+			StreamFetcher streamScheduler = streamFetcherFactory.make(broadcast, scope, schedulingService);
```

Comment:
My comment for previoud PR is still valid. Look at here: https://stackoverflow.com/a/36113562/538408
No need a streamFetcherFactory instance.